packages/flutter_markdown/lib/flutter_markdown.dart

```diff
@@ -0,0 +1,8 @@
+library flutter_markdown;
+
+import 'package:markdown/markdown.dart' as md;
+import 'package:flutter/material.dart';
+import 'package:string_scanner/string_scanner.dart';
+
+part 'src/markdown.dart';
+part 'src/syntax_highlighter.dart';
```

Comment:
The trouble with `part` is that it doesn't scale well to large libraries.  You can't see the dependencies between files and every file can see the private methods and fields from every other file.  For that reason, we've decided to adopt the style in Flutter of not using `part`.